extensions/wikia/AdEngine/js/utils/env.js

```diff
@@ -0,0 +1,13 @@
+define('ext.wikia.adEngine.utils.env', [
+	'wikia.window'
+], function (win) {
+	'use strict';
+
+	function isDevEnvironment() {
+		return /.+wikia-dev\.com$/.test(win.location.host);
```

Comment:
This will make `nandy.wikia-dev.com` and `nandywikia-dev.com` pass, won't it? I think we can be more strict and just allow for `nandy.wikia-dev.com`. What do you think?